internal/export/exportfile_test.go

```diff
@@ -0,0 +1,66 @@
+package export
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/exposure-notifications-server/internal/pb/export"
+	"github.com/stretchr/testify/assert"
+)
+
+const sampleFilename = "../../examples/export/testExport-2-records-1-of-1.zip"
```

Comment:
optional: You might consider restructuring this to be a test of marshalling followed by unmarshalling and decouple from the example file. Or if we find it specifically beneficial to use a file, you could put one in ./testdata

Reply:
Same thoughts here about putting it in ./testdata, thanks @gurayAlsac.